webknossos/examples/write_images_to_harddrive.py

```diff
@@ -0,0 +1,22 @@
+from pathlib import Path
+
+from webknossos import Dataset
+
+INPUT_FOLDER = Path("path/to/image_files")
```

Comment:
Please add a comment about the assumption of the input data:
- raw microscopy data without any segmentation layer
- just one directory with many files
- file naming contains z-order, e.g. `retina_scan_z0.jpeg, retina_scan_z1.jpeg`
- same resolution, ...